translate/tools/pocount.py

```diff
@@ -240,35 +242,38 @@ def footer(self):
         pass
 
 
+@dataclass
 class CsvRenderer(Renderer):
+    def __post_init__(self):
+        self._fields = {
+            "title": "Filename",
+            "translated": "Translated Messages",
+            "translatedsourcewords": "Translated Source Words",
+            "translatedtargetwords": "Translated Target Words",
+            "fuzzy": "Fuzzy Messages",
+            "fuzzysourcewords": "Fuzzy Source Words",
+            "untranslated": "Untranslated Messages",
+            "untranslatedsourcewords": "Untranslated Source Words",
+            "total": "Total Message",
+            "totalsourcewords": "Total Source Words",
+            "review": "Review Messages",
+            "reviewsourcewords": "Review Source Words",
+        }
+        self._file = io.StringIO()
+        self._writer = csv.DictWriter(self._file, self._fields.values())
```

Comment:
Wouldn't be better to use `sys.stdout` instead of buffering content in `StringIO`?

Reply:
Nice idea, updated code.